### DISCUSSION TITLE
Reactivate CKAN user or create a new user with an used email

### DISCUSSION THREAD

**Discussion by dbfz** [ckan/ckan#7038] [Q&A] [answered]
Hello together,

we use CKAN 2.9.5 under latest Ubuntu LTS release. I try to reactivate a user which is deleted or disabled. 

Today in the morning I created an account in the command line via "ckan -c ckan.ini sysadmin add ..." but it generated a random username. Possibly I made a mistake in the command. So I thought that I can delete the user and create it again. But now I get the message "None - {'email': ["The email address 'max.mastermann@mydomain.de' belongs to a registered user."]}"

So what can I do to create a new user with a used email address? 

Best regards
Kevin

**Comment by dbfz** [comment]
The only way for me actually is to delete the old user in the database table "user". This is the wrong way I now, but at moment I see no other solution.

**Comment by TomeCirun** [comment, marked as answer]
https://github.com/ckan/ckan/pull/6916 this might help, we fixed this issue on CKAN 2.10
